libxw/requirements.txt

```diff
@@ -0,0 +1,3 @@
+# https://pip.pypa.io/en/stable/user_guide/#requirements-files
+
+--editable libxw[dev]
```

Comment:
This means
* `--editable`: install in such a way that we can edit the source code and not need to re-install
* `[dev]`: install all the "dev" dependencies (stuff that's useful to have while developing, but end users -- if they existed -- wouldn't necessarily need)